src/resources/CaseAssist/tests/CaseAssist.spec.ts

```diff
@@ -0,0 +1,102 @@
+import API from '../../../APICore';
+import {New} from '../../BaseInterfaces';
+import CaseAssist from '../CaseAssist';
+import {CaseAssistModel, DocumentSuggestionsStrategies, TypingAidsStrategies} from '../CaseAssistInterfaces';
+
+jest.mock('../../../APICore');
+
+const APIMock: jest.Mock<API> = API as any;
+
+describe('CaseAssist', () => {
+    let caseAssist: CaseAssist;
+    const api = new APIMock() as jest.Mocked<API>;
+
+    const caseAssistModels: CaseAssistModel[] = [
+        {
+            id: 'CaseAssist-to-update-id-1',
+            name: 'New CaseAssist1',
+            documentSuggestionConfiguration: {
+                pipeline: 'fake_pipeline1',
+                fieldsToFeed: ['field1', 'field2'],
+                filter: 'filter_string',
+                strategy: DocumentSuggestionsStrategies.ITD,
+            },
+            typingAidsConfiguration: {
+                values: ['value1', 'value2'],
+                strategy: TypingAidsStrategies.StaticValues,
+            },
+        },
+        {
+            id: 'CaseAssist-to-update-id-2',
+            name: 'New CaseAssist2',
+            documentSuggestionConfiguration: {
+                pipeline: 'fake_pipeline1',
+                fieldsToFeed: ['field1', 'field2'],
+                filter: 'filter_string',
+                strategy: DocumentSuggestionsStrategies.ITD,
+            },
+            typingAidsConfiguration: {
+                filter: 'filter_string',
+                field: 'field_string',
+                strategy: TypingAidsStrategies.ValuesFromIndex,
+            },
+        },
+    ];
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+        caseAssist = new CaseAssist(api);
+    });
+
+    describe('list', () => {
+        it('should make a GET call to the CaseAssists base url', () => {
+            caseAssist.list({filter: 'foo', page: 2, pageSize: 10});
+
+            expect(api.get).toHaveBeenCalledTimes(1);
+            expect(api.get).toHaveBeenCalledWith(`${CaseAssist.baseUrl}?filter=foo&page=2&pageSize=10`);
```

Comment:
filter query parameter does not exist.

Reply:
Ah ok. In the mockups there's a text filter. I'll remove it then